apps/android/boot.js

```diff
@@ -197,6 +197,14 @@
           lastSteps = steps;
         }, event.int*1000);
       },
+      "fetchRec": function() {
+        // TODO: Loop from last fetched log up to current date and through the whole alphabet for the last letter.
+        // TODO: Decide on what names keys should have.
+        let lastLog = event.lastFetchedLog;
+        let log = require("Storage").open("recorder.log"+lastLog+".csv","r"); // TODO: Change to loop variable.
+        let logString = log.read(log.getLength());
```

Comment:
Is the idea here that it will load the entire log file in one go into logString?

I'm afraid that's not going to work as logs could be up to nearly 8MB in size, far more than fits in the ~200k of available RAM - you'll have to write it out line-by-line.

Reply:
Thanks!